src/rpc/RPCHelpers.cpp

```diff
@@ -405,61 +407,111 @@ getStartHint(ripple::SLE const& sle, ripple::AccountID const& accountID)
     return sle.getFieldU64(ripple::sfOwnerNode);
 }
 
+// traverse account's nfts
+// return Status if error occurs
+// return [nextpage, count of nft already found] if success
 std::variant<Status, AccountCursor>
-traverseOwnedNodes(
```

Comment:
We don't call this function except in unittests.

Reply:
If it's meant to only be used in unit tests and if we still need it.. I suggest we move it to some helper file in unittests folder instead.

Reply:
In unittest, we want to test it. since we don't need it, i port the test to new function